qiime/split_libraries_lea_seq.py

```diff
@@ -565,8 +555,8 @@ def get_consensus_seqs_lookup(random_bc_lookup,
                 # otherwise call get_consensus function
                 if fwd_cluster_ratio == 0 or rev_cluster_ratio == 0:
                     consensus_seq = "No consensus"
-                elif (fwd_cluster_ratio < max_cluster_ratio
-                        and rev_cluster_ratio < max_cluster_ratio):
+                elif (fwd_cluster_ratio > max_cluster_ratio
```

Comment:
Just found the place where this is tested below. I should have read through your unit tests more carefully first, sorry about that!